pandas/io/parsers.py

```diff
@@ -3600,15 +3600,15 @@ def __init__(self, f, colspecs, delimiter, comment, skiprows=None, infer_nrows=1
         self.comment = comment
         if colspecs == "infer":
             self.colspecs = self.detect_colspecs(
-                infer_nrows=infer_nrows, skiprows=skiprows
+                infer_nrows=infer_nrows, skiprows=skiprow
```

Comment:
You accidentally deleted a ``s``` here:)

```suggestion
                infer_nrows=infer_nrows, skiprows=skiprows
```

Reply:
I did, sry :(
I made another PR with the fix, but it failed

Ps: This is my first time contributing!